src/main/java/uk/gov/hmcts/reform/idam/health/probe/ScheduledHealthProbeIndicator.java

```diff
@@ -29,16 +29,22 @@ public ScheduledHealthProbeIndicator(
         this.failureHandling = failureHandling;
         this.freshnessInterval = freshnessInterval;
         this.clock = Clock.systemDefaultZone();
-        taskScheduler.scheduleWithFixedDelay(() -> refresh(), checkInterval);
+        taskScheduler.scheduleWithFixedDelay(this::refresh, checkInterval);
     }
 
     @Override
     public boolean isOkay() {
         if (status == Status.UNKNOWN) {
             return this.healthProbe.probe();
         }
-        return status == Status.UP
-                && LocalDateTime.now(clock).isBefore(statusDateTime.plus(freshnessInterval, ChronoUnit.MILLIS));
+
+        if (failureHandling == HealthProbeFailureHandling.MARK_AS_DOWN){
+            return status == Status.UP
+                    && LocalDateTime.now(clock).isBefore(statusDateTime.plus(freshnessInterval, ChronoUnit.MILLIS));
+        } else {
+            log.warn("{}: status evaluation ignored for this type of probe. failureHandling: {}", this.healthProbe.getName(), failureHandling.toString());
+            return true;
```

Comment:
I think it's worth a testing for this case.

Reply:
@kremi, new test was added